### PR TITLE
Use a higher store number to prevent segfaults

### PR DESCRIPTION
**Pull request by Wizek**
I've just recently started to try to use jsaddle-warp, and was running into segfaults, like so:

```
cabal: repl failed for exe:test-dev from test-0.17.0. The build process
segfaulted (i.e. SIGSEGV).
```

After some digging, it turned out that my app's foreign-store and your jsaddle-warp's store chose the same `Store` number which lead to the above crash.

I didn't test the code in this pull request yet, it's more intended as pseudo-code that may compile, and a way to ask if you are open to fixing this issue this way. Are you? Or would you have a different fix in mind?

### CODE REVIEW

**Comment by hamishmack** [comment]
Fine with me.  Let me know when it is safe to merge.

**Comment by Wizek** [comment]
@hamishmack ~So, since you've pinged me, I've attempted to verify this fix idea, and I am running into something strange.~

~The basic original segfault was, I imagined (and still seems the most likely explanation) that both my code and jsaddle-warp's code try to register a foreign-store entry at key 0, and get surprised (in a segfault way) when one of these try to read what the other wrote there. When I increased the store number in my app, the segfault went away.~

~Now I've built my app with this patch included, and when I set my app's store to 0 it still segfaults. At 10 it doesn't. And at 354 it still doesn't. Very strange, wondering if I'~

Scratch all that! I only swapped jsaddle out and not jsaddle-warp `(-.-)''`. After rectifying that oversight, all builds and works as expected.

So, long story short, yes, I'm okay with this being merged. And sorry for the delays.
